imap_processing/ialirt/l0/parse_mag.py

```diff
@@ -568,6 +568,19 @@ def process_packet(
     pkt_counter = get_pkt_counter(accumulated_data["mag_status"])
     accumulated_data["pkt_counter"] = pkt_counter
 
+    # Convert from incrementing uint16 (0-65535) to radians.
+    sc_spin_phase_rad = accumulated_data["sc_spin_phase"].astype(float) * (
+        2 * np.pi / 65535.0
```

Comment:
All of these should be 65536 since we have 2**16 possible values in the range I think.
```suggestion
        2 * np.pi / 65536.0
```

Reply:
Actually, I think it will depend on which variables you are wanting to do the math over. Latitude you do want to go [-pi, pi] inclusive, but I think here for the longitude you would want to exclude the final point because it is a duplicate [0, 2pi). Maybe it is easiest/best to directly use the conversion factors the MOC provided rather than calculate it here?